gce/startup-script.sh

```diff
@@ -32,20 +32,21 @@ service google-fluentd restart &
 apt-get update
 apt-get install -yq ca-certificates git build-essential supervisor
 
+
+# Fetch source code
+export HOME=/root
```

Comment:
question: Is this variable used anywhere? I don't see signs of $HOME being referenced, but I do see code reassigning the value of a HOME variable when this code is run.

Reply:
it's used by git. the prior note explained why it was there; I copied this comment over from python as I was trying to align the two samples more.